server/src/test/java/com/hedera/block/server/persistence/storage/read/BlockAsDirReaderTest.java

```diff
@@ -241,7 +241,7 @@ private void removeBlockItemReadPerms(
     // IOException while allowing the real setPerm() method to remain protected.
     private static final class TestBlockAsDirReader extends BlockAsDirReader {
         public TestBlockAsDirReader(PersistenceStorageConfig config) {
-            super(config, FileUtils.defaultPerms);
+            super(config, DEFAULT_DIR_PERMISSIONS);
```

Comment:
Either accept _private_ defaults, or set explicit values in the test.  Having a test explicitly set a value it doesn't control is asking for a brittle test failure.

Reply:
Agreed about that. Based on comments above and on what we shall agree on, if to remove or not the `filePerms` field in the super, this would either become `null` and the actual reader will use the overloaded method internally, or I will supply them by locally having them declared here and not use from the `common` module